src/utils/reported_error_utils.py

```diff
@@ -0,0 +1,5 @@
+from fastapi import HTTPException, status
+
+
+def raise_reported_error_not_found():
```

Comment:
W sumie potrzebny na jedną funkcję oddzielny plik?

Reply:
Sam tak zrobiłem - przy następnej pr'ce to zmienie, chyba, że sam chcesz to teraz machnąć